scripts/stac/imagery/collection.py

```diff
@@ -83,13 +83,15 @@ def __init__(
 
         self.add_providers(providers)
 
-    def add_capture_area(self, polygons: List[shapely.geometry.shape], target: str) -> None:
+    def add_capture_area(self, polygons: List[shapely.geometry.shape], target: str, artifact_target: str = "/tmp") -> None:
         """Add the capture area of the Collection.
         The `href` or path of the capture-area.geojson is always set as the relative `./capture-area.geojson`
 
         Args:
             polygons: list of geometries
-            target: path of the capture-area-geojson file
+            target: path where the capture-area.geojson file is saved
```

Comment:
"is saved" might be confusing, as it could indicate that it should already be there when the user runs this function. It might be useful to write it from the perspective of before running the function, for example, "path where the capture-area.geojson file will be saved" or "path of the new capture-area.geojson file".

"path" also could be confusing, since the target could also be an S3 URL. Maybe "location" would be a better word?